AopCaching.UnitTests/AopCacheTests.cs

```diff
@@ -71,7 +71,7 @@ public async Task TestNamedCache1Async()
             Assert.AreEqual(0, cache2.Misses);
 
             var service = new Service2();
-            var result = await service.MethodToCache1Async();
+            var result = await service.MethodToCache1Async().ConfigureAwait(false);

```

Comment:
Why does test code need this? - Please read what this does.

Reply:
removed from all unit tests